core/types/l2trace.go

```diff
@@ -60,7 +60,7 @@ type ExecutionResult struct {
 
 	// Record all accounts' state which would be affected AFTER tx executed
 	// currently they are just `from` and `to` account
-	AccountsAfter []*AccountWrapper `json:"accountAfter"`
+	AccountsAfter []*AccountWrapper `json:"accountAfter,omitempty"`
```

Comment:
I think we can keep "removing omitempty" here in case it may break roller.

Reply:
changed.